crates/miden-lib/asm/kernels/transaction/lib/epilogue.masm

```diff
@@ -238,7 +238,7 @@ end
 proc.execute_auth_procedure
     padw padw padw push.0.0.0
     # auth procedure is at index 0 within the account procedures section so no offsetting is needed.
-    exec.memory::get_acct_procedures_section_ptr
+    push.0 exec.memory::get_acct_procedure_ptr
```

Comment:
I would probably update the comment here since "no offsetting is needed" part is a bit confusing now.